data/dpkg_templates/bulk-extractor-control

```diff
@@ -0,0 +1,12 @@
+Source: bulk-extractor
+Section: utils
+Priority: optional
+Maintainer: Simson L. Garfinkel <slgarfin@nps.edu>
+Build-Depends: debhelper (>= 9), dh-autoreconf, autoconf, automake, flex, libtool, default-jdk, md5deep, openssl, bison, libssl-dev, zlib1g-dev, libxml2-dev
```

Comment:
Are md5deep and openssl really necessary build dependencies? or should these be in `Depends` instead?